ci/request-reviews/request-reviews.sh

```diff
@@ -60,10 +60,9 @@ git -C "$tmp/nixpkgs.git" remote add fork https://github.com/"$prRepo".git
 git -C "$tmp/nixpkgs.git" config remote.fork.partialclonefilter tree:0
 git -C "$tmp/nixpkgs.git" config remote.fork.promisor true
 
-# This should not conflict with any refs in Nixpkgs
-headRef=refs/remotes/fork/pr
-# Only fetch into a remote ref, because the local ref namespace is used by Nixpkgs, don't want any conflicts
-git -C "$tmp/nixpkgs.git" fetch --no-tags fork "$prBranch":"$headRef"
+# Our local branches mirror Nixpkgs, so make sure to not try to update any to avoid conflicts
+git -C "$tmp/nixpkgs.git" fetch --no-tags fork "$prBranch"
+headRef=$(git -C "$tmp/nixpkgs.git" rev-parse HEAD)
```

Comment:
IIUC, `headRef` should point to the PR head, which is not the case here.  The following patch should fix it.
```suggestion
headRef=$(git -C "$tmp/nixpkgs.git" rev-parse refs/remotes/"$prBranch")
```

---

BTW, I think there is no conflicts to avoid because only refs under "refs/remotes/fork" will be updated.

Reply:
Race conditions are the worst! I'd be glad to merge the change you suggest, @jian-lin, with a little test like what Silvan has illustrated in the PR headmatter above.

Reply:
I created https://github.com/NixOS/nixpkgs/pull/348004 :)